{"name": "Проверка логина с невалидным логином", "status": "passed", "start": 1717175467714, "stop": 1717175467890, "uuid": "0dc24c03-ac1f-42e7-bf9a-fdbd088e56fb", "historyId": "6d0f5ae368ae2abd7d65fe309158a601", "testCaseId": "6d0f5ae368ae2abd7d65fe309158a601", "fullName": "tests.test_login_courier.TestLoginCourier#test_courier_login_with_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "lisk"}, {"name": "thread", "value": "6457-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}